packages/react-devtools-extensions/edge/test.js

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const edge = require('windows-edge');
```

Comment:
This NPM package only supports the Windows platform, which means that most of us (myself included) can't actually use it 😄 I wasn't able to find a Mac compatible version of this when I last looked. Wondering if you know of any?

Reply:
Sorry about that. 

I found [this package ](https://www.npmjs.com/package/open) which might work with some code changes. I will find a mac to test and submit a new PR once ready.

Reply:
Excellent! Thank you!